katcp/resource_client.py

```diff
@@ -735,6 +735,208 @@ def issue_request(self, *args, **kwargs):
         """
         return self._client.wrapped_request(self.name, *args, **kwargs)
 
```

Comment:
Not sure I understand clearly: 
Are you referring to the removal of the abstract method set_sensor_strategy from class KATCPResource to KATCPClientResource? In which case I can return that as and abstract method as I came up with an alternate solution that kept the footprint the same on the resource client and the container.

Or are you referring to moving the ClientGroup?  The reason was that ClientGroup operators on KATCPClientResource and not on KATCPResource directly. In which case one often had to go back to resource to align ClientGroup after changing something in resource_client. This just seemed akward and unneccessary.